app/controllers/plantings_controller.rb

```diff
@@ -5,16 +5,19 @@ class PlantingsController < ApplicationController
   # GET /plantings
   # GET /plantings.json
   def index
-    @owner = Member.find_by(slug: params[:owner])
-    @crop = Crop.find_by(slug: params[:crop])
+    @owner = Member.find_by(slug: params[:owner]) if params[:owner]
+    @crop = Crop.find_by(slug: params[:crop]) if params[:crop]
     @plantings = if @owner
-                   @owner.plantings.includes(:owner, :crop, :garden).paginate(page: params[:page])
+                   @owner.plantings
                  elsif @crop
-                   @crop.plantings.includes(:owner, :crop, :garden).paginate(page: params[:page])
+                   @crop.plantings
```

Comment:
I think we're talking at cross-purposes. Here's what I meant to say:

- The code here matches the existing behaviour.
- That means we can filter by owner *or* crop, but not both.
- Maybe it would be useful to filter by both.
- That's out of scope for this PR, though.
- So the code above is fine.